osu.Game/Screens/Edit/Compose/Components/Timeline/ZoomableScrollContainer.cs

```diff
@@ -66,8 +66,9 @@ public float MinZoom
 
                 minZoom = value;
 
-                if (Zoom < value)
-                    Zoom = value;
+                // ensure zoom range is in valid state before updating zoom.
+                if (MinZoom < MaxZoom)
+                    Zoom = Zoom;
```

Comment:
Really don't like this. Can we make an `updateZoom` method or something similar instead?

Reply:
`updateZoom(Zoom)`? Sounds fine.

Reply:
Or have the parameter nullable / default null and call it without a parameter in these cases.